sormas-cargoserver/build_deploy.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+mvn cargo:stop
+
+# recompile
+cd ../sormas-base || exit
+
+if [ "$1" = "clean" ]; then
```

Comment:
Maybe we should rename this to "wipe", so that it is more clear the database will be dropped in this step

Reply:
yes